src/Microsoft.AspNet.Authentication.OAuth/project.json

```diff
@@ -4,7 +4,8 @@
     "dependencies": {
         "Microsoft.AspNet.DataProtection": "1.0.0-*",
         "Microsoft.AspNet.Authentication": "1.0.0-*",
-        "Microsoft.Framework.NotNullAttribute.Internal": { "type": "build", "version": "1.0.0-*" }
+        "Microsoft.Framework.NotNullAttribute.Internal": { "type": "build", "version": "1.0.0-*" },
+        "Newtonsoft.Json": "6.0.6"
```

Comment:
The handler uses Json.Linq yeah, we were getting it for free from RequestContainer before which is DOA